src/main/java/io/github/lucasstarsz/fastj/math/Maths.java

```diff
@@ -136,4 +136,17 @@ public static float magnitude(Point p) {
     public static boolean floatEquals(float a, float b) {
         return Math.abs(a - b) < FloatPrecision;
     }
+
+    /**
+     * Returns the value of lerping between x and y, by the interpolation point t.
```

Comment:
Avoid saying `returns` as the first word in your summary. This is more of a stylistic thing, but I recommend you to use the adjective of what the function does. 
For example: `Linearly interpolates between values {@code x} and {@code y}, by the interpolation point {@code t}.`

Secondly on your documentation: it's very good, but for anyone who doesn't understand what interpolation is, it might be best to provide a link to a resource that explains what it is and how it works. However, we can look for one at a later time.